### PR TITLE
Part 4 style

### PR DESCRIPTION
**Pull request by SamanAtashi**
# Bookstore: styling
Features introduced are as follows:

- styling using tailwind css

### CODE REVIEW

File: src/components/Books/Book/Book.js [hunk 1]
```diff
@@ -34,15 +38,76 @@ const Book = () => {
...
+          CHAPTER 12
+        </p>
+        <button
+          className="font-extralight min-w-max px-3 md:w-4/6 lg:w-3/6 h-8 border border-blue-600 rounded-md bg-blue-600 text-white text-sm cursor-pointer hover:bg-blue-500"
+          type="button"
+        >
+          UPDATE PROGRESS
+        </button>
+      </div>
     </li>
```
Comment (DeliceLydia) [inline src/components/Books/Book/Book.js]: - [x] There is some red line on your page please remove them
bookstore
- [x] I suggest that you use the same color used on the design maybe this `#0290ff`

File: src/components/Navigation/Navigation.js [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
 import { Link } from 'react-router-dom';
 
 const Navigation = () => (
-  <nav>
-    <Link to="/">Books</Link>
-    <Link to="/categories">Categories</Link>
+  <nav className="bg-white w-full flex justify-between items-center px-14 py-6 shadow-md">
+    <div className="flex items-center space-x-7">
+      <h2 className="text-blue-600  font-bold text-3xl">
+        Welcome to BookStore
```
Comment (DeliceLydia) [inline src/components/Navigation/Navigation.js]: - [x] As the requirements said you need to make your page exactly like the design so here use `BOOKSTORE CMS`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @SamanAtashi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @SamanAtashi, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

Well-Done-Applause


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**